pkg/config/config_template.yaml

```diff
@@ -4149,10 +4149,12 @@ api_key:
   ##
   ## Example:
   ## - conditions:
-  ##     service_name: exampleService
+  ##     command_line_regex: command_line_regex: "exampleService"
   ##   configuration:
-  ##     version: 1
   ##     service_language: dotnet
+  ##     dd_env: staging
+  ##     dd_service: exampleService
+  ##     dd_version: 1.2.3
```

Comment:
would it be possible to include all of the environment variables we currently support?